docs/terminal/melbourne.md

```diff
@@ -32,45 +33,56 @@ See also: [AV ADC Offline](#av-adc-offline).
 !!! tip
     When AV ADC is not online, consider publishing an **ATIS Zulu** for the aerodrome, to inform pilots about the airspace reclassification. The *More ATIS* plugin has a formatted Zulu ATIS message.
 
+### Avalon Approach (MAV)
+MAV is responsible for the Class D and E airspace in the AV CTR, `SFC`-`A045`.  
+They are responsible for the same portion of airspace when it is [reclassified](#av-ctr) due to AV ADC being offline.
+
+The Southern Wedge of MAV (shown below) is Class E `A015`-`A045`, and extends in to YWE(WON) airspace. YWE(WON) is responsible for the airspace **above and below** it, and it is **Class G**.
+
+<figure markdown>
+![MAV Southern Wedge](img/mavsouth.png){ width="500" }
+  <figcaption>MAV Southern Wedge</figcaption>
+</figure>
+
 ### Airspace Division
-The divisions of the airspace between **MAE**, **MDN**, and **MDS** change based on the Runway Mode.
+The divisions of the airspace between **MAE**, **MDN**, **MDS**, and **MAV** change based on the Runway Mode.
 
 !!! note
     The following diagrams do not include non ML TCU areas of responsibility such as MB CTR or AV ADC
 
 #### 09
 <figure markdown>
-![09 TCU Structure](img/ML09annotated.png){ width="700" }
+![09 TCU Structure](img/ml09.png){ width="700" }
   <figcaption>09 TCU Structure</figcaption>
 </figure>
 
 #### 09A16D
 <figure markdown>
-![09A16D TCU Structure](img/ML09A16Dannotated.png){ width="700" }
+![09A16D TCU Structure](img/ml09a16d.png){ width="700" }
   <figcaption>09A16D TCU Structure</figcaption>
 </figure>
 
 #### 16 / 16A27D
 <figure markdown>
-![16 / 16A27D TCU Structure](img/ML16annotated.png){ width="700" }
+![16 / 16A27D TCU Structure](img/ml16.png){ width="700" }
   <figcaption>16 / 16A27D TCU Structure</figcaption>
 </figure>
 
 #### 27 / 27AD34D
 <figure markdown>
-![27 / 27AD34D TCU Structure](img/ML27annotated.png){ width="700" }
+![27 / 27AD34D TCU Structure](img/ml27.png){ width="700" }
   <figcaption>27 / 27AD34D TCU Structure</figcaption>
 </figure>
 
 #### 34
 <figure markdown>
-![34 TCU Structure](img/ML34annotated.png){ width="700" }
+![34 TCU Structure](img/ml34.png){ width="700" }
   <figcaption>34 TCU Structure</figcaption>
 </figure>
 
 ## Departure Procedures
 ### ML (RADAR) SID
-The **ML (RADAR) SID** is used for all non-jet IFR departures (and certain jet aircraft) from YMML.  Unlike at other aerodromes, **ML ADC** does *not* need to coordinate these departures with a 'Next' call, provided they are assigned the standard assignable level and a [Standard Assignable Heading](#standard-assignable-departure-headings).
+The **ML (RADAR) SID** is used for all non-jet IFR departures (and certain jet aircraft) from YMML. Unlike at other aerodromes, **ML ADC** does *not* need to coordinate these departures with a 'Next' call, provided they are assigned the standard assignable level and a [Standard Assignable Heading](#standard-assignable-departure-headings).
```

Comment:
Sydney has standard assignable headings, would it make sense to remove the `Unlike at other aerodromes` text?

Reply:
Awareness of standard assignable headings has improved dramatically, so we can probably look to strip this section down now anyway. Thanks Glenn